source/developer/integration-faq.md

```diff
@@ -41,6 +41,48 @@ For self-hosted deployments in small setups you might host integrations on the s
 
 When self-hosting restrictions are less strict, AWS, Heroku and other public cloud options could also be used.
 
+## How to create a bot account without using webhooks
+
+Deployments that cannot create bot accounts via webhooks due to security reasons can use the following approach:
+
+1. Create a bot account using a secure email and strong password.
+2. Manually add the account to all teams and channels it needs access to. If your deployment has a lot of teams or channels, you may create a CLI script to automate the process.
+   - In a testing environment, you may also make the bot account a System Admin, giving the bot permissions to post to any channel. Not recommended in production due to potential security vulnerabilities.
+3. Provide the email and password to your integration, and store it in a secure location with restricted access.
+4. Have your integration use the email and password with an [`/api/v4/login`](https://api.mattermost.com/v4/#tag/authentication) endpoint to retrieve a session token. The session token is used to authenticate to the Mattermost system.
+   - Make an HTTP POST to `your-mattermost-url.com/api/v4/users/login` with a JSON body indicating the bot's email and password.
+  
+     ```
+     curl -i -d '{"login_id":"someone@nowhere.com","password":"thisisabadpassword"}' http://localhost:8065/api/v4/users/login
```

Comment:
I don't think we should use curl commands as examples. The bot is the one making the request, a curl command implies that a person is making the request from a terminal.  We can show the actual request as an example, like so:

```
POST /api/v4/users/login HTTP/1.1
Host: your-mattermost-url.com
Content-Length: 66
Content-Type: application/json

{"login_id":"someone@nowhere.com","password":"thisisabadpassword"}
```